src/index.ts

```diff
@@ -27,7 +27,7 @@ import type { Server } from '@modelcontextprotocol/sdk/server/index.js';
 export async function createConnection(userConfig: Config = {}, contextGetter?: () => Promise<BrowserContext>): Promise<Server> {
   const config = await resolveConfig(userConfig);
   const factory = contextGetter ? new SimpleBrowserContextFactory(contextGetter) : contextFactory(config.browser);
-  return mcpServer.createServer(new BrowserServerBackend(config, factory));
+  return mcpServer.createServer(new BrowserServerBackend(config, factory), false);
```

Comment:
make it configurable?

Reply:
Let's do that upon request. They can implement it on the call site if needed, same way we did.